docs/reference/es_compatible_api.md

```diff
@@ -169,7 +169,7 @@ following syntax.
 {
   // ...
   "sort" : [
-    { "timestamp" : {"order" : "asc"}},
+    { "timestamp" : {"format": "epoch_millis","order" : "asc"}},
```

Comment:
surprise..... I went for `epoch_millis_as_int` format as elasticsearch returns milliseconds as strings with format `epoch_millis`. Thus I added the ugly `as_int` to avoid confusion :///// 

```suggestion
    { "timestamp" : {"format": "epoch_millis_as_int","order" : "asc"}},
```